File: model/user_tracks.go
package model

import (
	"github.com/jinzhu/gorm"
)

type UserTracks struct {
	gorm.Model
	UserID  int64 `gorm: "type: bigint; primary_key; auto_increment: false; unique; not null"`
	TrackID int64 `gorm: "type: bigint; primary_key; auto_increment: false; unique; not null"`
}

File: admin/backend/src/db/model/user-tracks.ts
import { Model, snakeCaseMappers } from 'objection';

export class UserTracks extends Model {
    userId: string;
    trackId: string;
    
    static get tableName() {
        return 'user_tracks';
    }

    static get idColumn() {
        return ['user_id', 'track_id'];
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default UserTracks;